hw6/mergeSort/List.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <stdbool.h>
+
+// This is strust of list, it consists of links to previous and next elenents and size of list
```

Comment:
```suggestion
// This is struct of list, it consists of links to previous and next elements and size of list
```
Но вообще это раскрытие деталей реализации